pkg/build/util/util.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import (
+	"fmt"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+)
+
+// GetBuildPodName returns name of the build pod.
+func GetBuildPodName(build *buildapi.Build) string {
+	return fmt.Sprintf("pod-%s", build.Name)
```

Comment:
I'm interested in knowing the details, @smarterclayton can you elaborate please?